sdk/dotnet/Pulumi.Automation/PulumiFn.cs

```diff
@@ -18,7 +17,7 @@ internal PulumiFn()
         {
         }
 
-        internal abstract Task<ExceptionDispatchInfo?> InvokeAsync(IRunner runner, CancellationToken cancellationToken);
+        internal abstract Task<int> InvokeAsync(IRunner runner, CancellationToken cancellationToken);
```

Comment:
What does the int stand for, perhaps a quick comment here? Exit code?

Reply:
yea it should be the exit code returned from one of the `IRunner.RunAsync(...)` overloads. Depending on the shape of the inline function (generic, service provider, or delegate) it needs to use a different overload so `PulumiFn` needs to handle the actual call to `IRunner`. I can add a quick comment